spec/sidekiq/form1010cg/submission_job_spec.rb

```diff
@@ -3,13 +3,12 @@
 require 'rails_helper'
 
 RSpec.describe Form1010cg::SubmissionJob do
-  let(:claim) do
-    require 'saved_claim/caregivers_assistance_claim'
+  let(:claim) { create(:caregivers_assistance_claim, form:) }
+  let(:form) { VetsJsonSchema::EXAMPLES['10-10CG'].clone.to_json }
```

Comment:
🎨 I know it runs the `let`s as they're referenced, but having them in reverse order weirds me out a bit?
```suggestion
  let(:form) { VetsJsonSchema::EXAMPLES['10-10CG'].clone.to_json }
  let(:claim) { create(:caregivers_assistance_claim, form:) }
```
maybe just me?

Reply:
Yeah like you said it doesn't matter, but yeah sometimes having them in order that they are called can help with readability. I'll go ahead and update it.